tests/autoblocks/test_autoblocks_configs.py

```diff
@@ -0,0 +1,249 @@
+import json
+import os
+from unittest import mock
+
+import pydantic
+
+from autoblocks._impl.config.constants import API_ENDPOINT
+from autoblocks.configs.config import AutoblocksConfig
+from autoblocks.configs.models import DangerouslyUseUndeployedRemoteConfig
+from autoblocks.configs.models import DangerouslyUseUndeployedWithRevision
+from autoblocks.configs.models import LatestDangerouslyUseUndeployed
+from autoblocks.configs.models import LatestRemoteConfig
+from autoblocks.configs.models import RemoteConfigWithVersion
+from tests.util import MOCK_CLI_SERVER_ADDRESS
+
+
+class MyConfigValue(pydantic.BaseModel):
+    my_val: str
+
+
+class MyConfig(AutoblocksConfig[MyConfigValue]):
+    pass
+
+
+@mock.patch.dict(
+    os.environ,
+    {
+        "AUTOBLOCKS_API_KEY": "mock-api-key",
+    },
+)
+def test_gracefully_handles_parser_error(httpx_mock):
+    httpx_mock.add_response(
+        url=f"{API_ENDPOINT}/configs/my-config-id/versions/latest",
+        method="GET",
+        match_headers={"Authorization": "Bearer mock-api-key"},
+        json=dict(
+            id="my-config-id",
+            version="1",
+            value=dict(
+                random_val="val-from-remote",
+            ),
+        ),
+    )
+
+    config = MyConfig(
+        value=MyConfigValue(my_val="initial-val"),
+    )
+    config.activate_from_remote(
+        config=LatestRemoteConfig(id="my-config-id", latest=True), parser=MyConfigValue.model_validate
```

Comment:
should we default latest=True on `LatestRemoteConfig` so you don't have to repeat it? so it'd just be:

```python
config.activate_from_remote(
  config=LatestRemoteConfig(id="my-config-id"),
  ...,
)
```

since it's already in the name that it's latest